### PR TITLE
Fix aggregator timestamp check

### LINKED ISSUE #115: Aggregator timestamp check is wrong
https://github.com/semiotic-ai/timeline-aggregation-protocol/blob/341d0ab4232de7d5ccd366c201fd1b9798c81b54/tap_aggregator/src/aggregator.rs#L105

Should actually be `>=`.

### PR DESCRIPTION
**Pull request by aasseman**
Fixes #115

### CODE REVIEW

**Comment by aasseman** [comment]
PS: `cargo test` @ [3a36164](https://github.com/semiotic-ai/timeline-aggregation-protocol/pull/116/commits/3a3616466b906833b767394cbc34d325cbc117e2) should fail

**Review by severiano-sisneros — DISMISSED** [review]
Good catch. Fix, looks good to me. I confirmed that this fix matches our spec.

File: tap_aggregator/src/aggregator.rs [hunk 1]
```diff
@@ -194,71 +191,65 @@ mod tests {
...
-        allocation_ids: Vec<Address>,
-    ) {
-        let time = SystemTime::now()
-            .duration_since(UNIX_EPOCH)
-            .unwrap()
-            .as_millis() as u64;
+    async fn check_receipt_timestamps(keys: (LocalWallet, Address), allocation_ids: Vec<Address>) {
+        // Create receipts with consecutive timestamps
+        let mut receipts = Vec::new();
+        for i in 10..20 {
```
Comment (ColePBryan) [inline tap_aggregator/src/aggregator.rs:197]: It seems these numbers are used throughout the test. Could this be updated be more clear that the numbers are all related? Possible store this range in a variable and use `max()` / `min()` on the range where applicable.

**Review by ColePBryan — COMMENTED** [review]
The update is consistent with the core implementation. I just have a small comment on readability of the test.

**Comment by ColePBryan** [comment]
@severiano-sisneros could we also make sure to add a test to the integration tests to ensure this is consistent across the library? Seems like the best place to ensure the consistency.

**Comment by aasseman** [comment]
Rebased onto main + addressed comments

File: tap_aggregator/src/aggregator.rs (hunk 1 above)
Comment (aasseman) [reply tap_aggregator/src/aggregator.rs]: Fixed in [0eca077](https://github.com/semiotic-ai/timeline-aggregation-protocol/pull/116/commits/0eca077705b505b604b612d7a8c0dc64dc1baafd)

**Comment by ColePBryan** [comment]
Added #117 to confirm consistent behavior throughout the library.

**Pull request merged**